leftwm-core/src/handlers/command_handler.rs

```diff
@@ -238,8 +239,8 @@ fn toggle_state(state: &mut State, window_state: WindowState) -> Option<bool> {
     let toggle_to = !window.has_state(&window_state);
     let tag_id = state.focus_manager.tag(0)?;
 
-    if window_state == WindowState::Fullscreen {
-        //Going to fullscreen, so we save the window order
+    if window_state == WindowState::Fullscreen || window_state == WindowState::Maximized {
+        //Going to fullscreen/maximized, so we save the window order
```

Comment:
```suggestion
        // Going to fullscreen or maximized, so we save the window order
```
The spacing in the next line is fixed in `main`, but this would conflict.